src/test/lib/ApiTest.cpp

```diff
@@ -2654,6 +2654,26 @@ void QuicTestGlobalParam()
     }
 #endif
 
+    //
+    // QUIC_PARAM_GLOBAL_STATELESS_RESET_KEY
+    //
+    {
+        uint8_t statelessResetkey[QUIC_STATELESS_RESET_KEY_LENGTH - 1];
+        CxPlatRandom(sizeof(statelessResetkey), statelessResetkey);
+        TestScopeLogger LogScope0("QUIC_PARAM_GLOBAL_STATELESS_RESET_KEY");
+        {
+            // should the global setting hard-coded?
```

Comment:
What does this comment mean? And is it necessary?

Reply:
will remove it.